proto/ibc/lightclients/localhost/v1/localhost.proto

```diff
@@ -11,8 +11,6 @@ import "gogoproto/gogo.proto";
 message ClientState {
   option (gogoproto.goproto_getters) = false;
 
-  // unique chain identifier
```

Comment:
Just want to double check: since the localhost client is actually removed in v7, then this change is not API breaking, right?

Reply:
Yep, this is technically a new addition so not API breaking!